waiter/integration/waiter/token_request_test.clj

```diff
@@ -917,27 +917,27 @@
         (assert-response-status response 400)))
 
     (testing "can't use invalid token that's valid for zookeeper"
-      (let [response (make-request waiter-url "/pathabc" :headers {"X-Waiter-Token" "bad#token"})]
```

Comment:
also edit waiter/integration/waiter/busy_instance_test.clj and waiter/integration/waiter/streaming_test.clj?

just run `waiter$ grep -r X-Waiter .` ?

Reply:
Done.